packages/tools/pie-css/README.md

```diff
@@ -148,6 +176,133 @@ Some PIE Web Components use a z-index value to control how they stack on a webpa
 | --dt-z-index-cookie-banner | 5000           |
 | --dt-z-index-toast         | 6000           |
```

Comment:
just thinking - we might want to rename this or re-think in the future as top layer usage kind of negates any z-index here

Reply:
I won't tackle this as part of this PR (as out of scope to amend this). Let's chat about when you're in next if it's worth looking at.